v0/destinations/salesforce/transform.js

```diff
@@ -39,20 +41,20 @@ async function getSFDCHeader(destination) {
   )}${encodeURIComponent(destination.Config.initialAccessToken)}&client_id=${
     destination.Config.consumerKey
   }&client_secret=${destination.Config.consumerSecret}&grant_type=password`;
-  let response;
-  try {
-    response = await axios.post(authUrl, {});
-  } catch (error) {
-    logger.error(error);
+  const sfAuthResponse = await httpPOST(authUrl, {});
+  const processedsfAuthResponse = processAxiosResponse(sfAuthResponse);
+  if (processedsfAuthResponse.status !== 200) {
     throw new CustomError(
-      `SALESFORCE AUTH FAILED: ${error.message}`,
-      error.status || 400
+      `SALESFORCE AUTH FAILED: ${JSON.stringify(
+        processedsfAuthResponse.response
+      )}`,
+      processedsfAuthResponse.status
```

Comment:
Should it be?
```suggestion
      processedsfAuthResponse.status || 400
```

Reply:
`processAxiosResponse` response handles this there wont be a case where status would be undefined

Reply:
Cool